scraper/scrapers/svt.py

```diff
@@ -286,6 +294,27 @@ def check_end_page(from_, region, page):
 
     return response_list
 
+def check_in_page(time_to_check, news_list):
+    if check_time(news_list[FIRST], AFTER, time_to_check) and check_time(news_list[LAST], BEFORE, time_to_check):
```

Comment:
return (check_time(news_list[FIRST], AFTER, time_to_check) and check_time(news_list[LAST], BEFORE, time_to_check))

Reply:
plz fix